### ISSUE TITLE
Add more numeric reponse codes

### ISSUE THREAD

**Issue by deavmi** [deavmi/birchwood#1]
New IRC standards have new numeric response codes that need to be added, else they are reported as "invalid" by the library. This needs to be added now.

No crashes happen, exceptions are caught such that they can be logged for now.

**Comment by deavmi** [comment]
@rany2 Big if true

**Comment by rany2** [comment]
@deavmi biggie smalls?

**Comment by deavmi** [comment]
Yes, Big Pappa

**Comment by deavmi** [comment]
We need more IRC RFC in this bruv :)

**Comment by deavmi** [comment]
Currently, we don't crash we catch a vonmversion to enum error:

```
std/conv.d(2111): Value (333) does not match any member value of enum 'ReplyType'
----------------
/usr/include/dmd/phobos/std/conv.d:2111 pure @safe birchwood.protocol.constants.ReplyType std.conv.toImpl!(birchwood.protocol.constants.ReplyType, ulong).toImpl(ulong) [0x90c58e]
/usr/include/dmd/phobos/std/conv.d:224 pure @safe birchwood.protocol.constants.ReplyType std.conv.to!(birchwood.protocol.constants.ReplyType).to!(ulong).to(ulong) [0x90bc1c]
../../../../.dub/packages/birchwood-1.0.1/birchwood/source/birchwood/protocol/messages.d:109 birchwood.protocol.messages.Message birchwood.protocol.messages.Message.__ctor(immutable(char)[], immutable(char)[], immutable(char)[]) [0x908c7f]
../../../../.dub/packages/birchwood-1.0.1/birchwood/source/birchwood/protocol/messages.d:232 birchwood.protocol.messages.Message birchwood.protocol.messages.Message.parseReceivedMessage(immutable(char)[]) [0x9092bd]
../../../../.dub/packages/birchwood-1.0.1/birchwood/source/birchwood/client/receiver.d:204 void birchwood.client.receiver.ReceiverThread.recvHandlerFunc() [0x90847a]
??:? void core.thread.context.Callable.opCall() [0xbedda4]
??:? thread_entryPoint [0xbed882]
??:? [0x7fd5744ae12c]
??:? [0x7fd57452fbbf]
```

**Comment by deavmi** [comment]
I should probably work on this, let me know if you are keen on adding some more cool stuff to birchwood in terms of IRC support @rany2

**Comment by deavmi** [comment]
Marking as closed since #11 is done and #16 is in the works
